packages/presentation/lib/home_screen/screen/home_widget.dart

```diff
@@ -12,21 +16,37 @@ class HomeWidget extends StatefulWidget {
 }
 
 class _HomeWidgetState extends State<HomeWidget> {
-  final homeBloc = HomeBloc(PalindromeCaseImpl());
+  late final Dio _dio;
+  late final ApiService _apiService;
+  late final CancelToken _cancelToken;
+  late final NetworkRepository _palindromeRepository;
+  late final HomeBloc _homeBloc;
   String textResult = "";
   String myText = "";
 
+  @override
+  void initState() {
+    super.initState();
+    _dio = dioBuilder('https://fpwebservice.herokuapp.com');
+    _apiService = ApiService(
+      _dio,
+    );
+    _cancelToken = CancelToken();
+    _palindromeRepository = NetworkRepository(_apiService, _cancelToken);
+    _homeBloc = HomeBloc(PalindromeCaseImpl(_palindromeRepository));
+  }
```

Comment:
seems that u can initialize the variables in the place of their declaration, no?

Reply:
I can't initialize the variables, because i get error "The instance member can't be accessed in an initializer", but I moved code from initstate. 
<img width="535" alt="image" src="https://user-images.githubusercontent.com/99738581/158136385-af778f5f-52a6-44b3-b2d1-782fb9b06fd9.png">

Reply:
i moved initialize the variables in constructer 
<img width="645" alt="image" src="https://user-images.githubusercontent.com/99738581/158148394-3b955bb3-88b5-4ade-aa2f-1a2077de1b7f.png">